File: pizzeria-meal/src/main/java/pl/pizzeria/order/domain/OrderRequest.java
package pl.pizzeria.order.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.util.List;

@Getter
@AllArgsConstructor
@NoArgsConstructor
public class OrderRequest {
    private OrderDetails orderDetails;
    private List<MealRequest> mealRequests;
}

File: angular-app/src/app/model/order-request.ts
import { MealRequest } from "./meal-request";
import { OrderDetails } from "./order-details";

export class OrderRequest {
    orderDetails: OrderDetails;
    mealRequests: MealRequest[];

    constructor(orderDetails: OrderDetails, mealRequests: MealRequest[]) {
        this.orderDetails = orderDetails;
        this.mealRequests = mealRequests;
    }
}
